README.md

```diff
@@ -159,6 +165,67 @@ ember exam --split=3 --partition=3
 
 **Note 2**: _You must be using Testem version `1.5.0` or greater for this feature to work properly._
 
+### Test Load Balancing
+
+```bash
+$ ember exam --load-balance=<num>
+```
+
+`load-balance` allows you to specify a number of browser(s) to load test files against. It will order the test files by test type before doing load balancing. For example, if you specify load-balance equals 3, then three browser instances will be created and the output will look something like:
+
+```bash
+ok 1 Chrome 66.0 - Browser Id 1 - some test
+ok 2 Chrome 66.0 - Browser Id 2 - some another test
+ok 3 Chrome 66.0 - Browser Id 3 - some the other test
+```
+
+You can also specify the `split` and `partition` options with `load-balance` to load a portion of AMD modules to each machine.
+
+```bash
+$ ember exam --split=<num> --partition=<num> --load-balance=<num>
+```
+
+This command will split AMD modules and load-balance tests from the specified partition across the browsers. If you specify a split of 2, a partition of 1, and load-balance of 3 the AMD modules are split into 2 and the first half of the modules is loaded into three browsers. The output will look something like below:
+```base
+ok 1 Chrome 66.0 - Exam Partition 1 - browser Id 1 - some test
+ok 2 Chrome 66.0 - Exam Partition 1 - browser Id 2 - some another test
+ok 3 Chrome 66.0 - Exam Partition 1 - browser Id 3 - some the other test
+```
+
+If `load-balance` is used without specifying a number of browser(s), the option has a default value of 1 and must be passed last. Otherwise, any following arguments will be read as a value for `--load-balance`. For instance:
+
+```bash
+# don’t do this
+# The number of browser instances = --server --no-launch
+$ ember exam --load-balance --server --no-launch
+
+# do this
+$ ember exam --server --no-launch --load-balance
+```
+
+##### Test Failure Reproduction
+
+Since Ember Exam with `load-balance` loads modules one at a time the order of executing modules is not deterministic. For this reason a test execution with `load-balance` creates a json file to track which browser executes which modules in which order. To replay the tests for one particular browser, do the following:
```

Comment:
Since we mentioned the json file here, can we also add a sentence noting where the file gets created and the naming structure?

Also, a small nitpick: ` To replay the test execution for one particular browser`